.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,94 @@
+name: "Build Pipeline (Docker)"
+
+on:
+  push:
+    branches:
+      - jesse/pipeline-deploy
+      - main
+
+env:
+  IMAGE_NAME: "${{ vars.GAR_LOCATION }}-docker.pkg.dev/${{ vars.PROJECT_ID }}/${{ secrets.DOCKER_REPOSITORY }}/polygon-cli"
+
+  ATTESTOR_PROJECT_ID: ${{ vars.PROJECT_ID }}
+  KEY_RING: "jesse-artifacts-ring"
```

Comment:
Can we set up `KEY_RING`, `KEY` and `ATTESTOR` for the whole devtools team?

Reply:
yes this will happen once i get permission to the shared-dev repo